docs/application/tizen-studio-0/setup/windows.md

```diff
@@ -0,0 +1,172 @@
+2. Locate the **jdk-10** and the **javafx-sdk-11.0.2** directories after extraction.
+    > **Note**
+    >
+    > - The **jdk-10** is the sub directory under the **openjdk-10_windows-x64_bin** directory.
+    >- The **javafx-sdk-11.0.2** is the sub directory under the **openjfx-11.0.2_windows-x64_bin-sdk** directory.
+3. Double click to open the **javafx-sdk-11.0.2** directory.
+4. Copy all the files except the **src.zip** from the **lib** sub directory located in the **javafx-sdk-11.0.2** directory.
+5. Paste all the copied files in the **lib** sub directory of the  **jdk-10** directory.
+6. Copy all the files from the **bin** sub directory in the **javafx-sdk-11.0.2**.
+7. Paste all the copied files in the **bin** sub directory of the **jdk-10** directory.
+
+   >**Note**
+   >
+   >- In this installation guide, setting up OpenJDK is covered. However, Oracle JDK versions until version 10 are also supported.
+   >- For Oracle JDK 10 installation steps, see the [Oracle Documentation](https://docs.oracle.com/javase/10/install/installation-jdk-and-jre-microsoft-windows-platforms.htm#JSJIG-GUID-DAF345BA-B3E7-4CF2-B87A-B6662D691840).
+   >- If you have installed Oracle JDK 10 or below already, you can skip setting up OpenJDK. 
+ 
+### Set Up Open JDK Path 
+
+To set up the JDK path, follow these steps:
+
+1. On the Windows desktop, click **Start > Control Panel > System and security > System > Advanced system settings**, the System Properties window appears. 
+3. In the **Startup and Recovery** section, click **Environment Variables** the Environment Variables window appears.  
+4. Click **New** under **User variables for {user}** section, also in case the **JAVA_HOME** variable is already in the list, select it and click **Edit**, enter the following details: 
+	- Variable name: **JAVA_HOME** 
+	- Variable value: **C:\Path\to\your\openJDKtype**, i.e. browse for the JDK directory path (for example, C:\Users\user\Desktop\jdk-10.0.2) 	
+5. Click **OK**, the Environment Variable list is updated with the **JAVA_HOME** variable.
+6. Click **New** under the **System variables** section also in case, if the **Path variable** already exists, select it and click **Edit**, and enter the following details: 
+	- Variable name: **Path** 
+	- Variable value: **C:\Path \to\ openJDK\bin directory**; i.e. browse for the **bin** subdirectory path in the JDK directory  (for example, C:\Users\user\Desktop\jdk-10.0.2\bin ) 
+8. Click **OK**, the System Variable list is updated with the **Path** variable.
+9. Save and close the **Environment Variables** window.
+10. Open Command Prompt and type the `java –version` command to verify whether the OpenJDK version 10 is installed or not. Entering the command must give the following output: 
+
+   ```windows console
+   C:\Users\exampleconsole>java -version
+   openjdk version "10" 2018-03-20
+   OpenJDK Runtime Environment 18.3 (build 10+46)
+   OpenJDK 64-Bit Server VM 18.3 (build 10+46, mixed mode)
+   ```
```

Comment:
he title of menu tree

Reply:
Sorry, Ms.Kang, I could not understand your comment, can you please explain to me further on this comment, thanks.